engine/utils/cortex_utils.h

```diff
@@ -34,8 +34,10 @@ constexpr static auto kTensorrtLlmPath = "/engines/cortex.tensorrt-llm";
 
 inline std::string models_folder = "./models";
 inline std::string logs_folder = "./logs";
-inline std::string logs_base_name = "./logs/cortex";
+inline std::string logs_base_name = "./logs/cortex.log";
+inline std::string logs_cli_base_name = "./logs/cortex-cli.log";
 inline size_t log_file_size_limit = 20000000;  // ~20 mb
```

Comment:
@nguyenhoangthuan99 Is this variable still used?

Reply:
for now, we don't use the file size limit anymore. I will update it